lib/Containers/MerkleTree.cpp

```diff
@@ -1002,7 +1297,7 @@ void MerkleTree<Hasher, BranchingBits>::modify(std::vector<std::uint64_t> const&
 
 template <typename Hasher, std::uint64_t const BranchingBits>
 bool MerkleTree<Hasher, BranchingBits>::modifyLocal(
-    Node& node, std::uint64_t count, std::uint64_t value, bool isInsert) noexcept {
+    Node& node, std::uint64_t count, std::uint64_t value, bool isInsert) {
```

Comment:
Why is this no longer noexcept? I do not see any exception which could happen.

Reply:
I understand the cases below, where `node(i)` is called, but here, it seems `noexcept`.

Reply:
True. Changed it back to be `noexcept`.